jenkins-admin/managed-config-files/custom/clouds-yaml/customizedCredentialMappings.sandbox.yaml

```diff
@@ -0,0 +1,4 @@
+---
+customizedCredentialMappings:
+  - credentialsId: "dummy"
```

Comment:
This should be the name of the actual credentials ID created for the "CLOUD" token. I think we don't have these yet, so you will need to create them from scratch as a username/password type credential.. you can name is something like "clouds-auth-credentials" or simiar